.prettierrc.json

```diff
@@ -1,6 +1,9 @@
 {
-    "trailingComma": "none",
-    "tabWidth": 4,
+    "tabWidth": 2,
```

Comment:
selv synes jeg 4 gjør det mye mer leselig, men dette er selvfølgelig en smakssak

Reply:
Der er jeg nok uenig. Men det er jo en evig diskusjon 😅
Vi får se om noen flere slenger seg på diskusjonen

Reply:
@egdegauperaa har nok en del å komme med her :P

Reply:
Jeg er enig med Bergljot for hva det er verdt! :D

Reply:
> Jeg er enig med Bergljot for hva det er verdt! :D

Spesielt siden de fleste av JetBrains sine IDE'er kommer med 4 som standard, feks IntelliJ, VS2022(?), Rider...

Reply:
For hva det er verdt er jeg enig med Bergljot... De fleste IDE'er kommer med 4 tabWidth som standard. 😄

Reply:
2 = yaml
4 = alt annet

mine $0.02

Reply:
Én tab i Java.
Neida, er egentlig helt likegyldig til hvor mange, så lenge det er likt i alle filene per repo.
Jeg har hørt om noen som ville ha 1 tab og 1 space på alle linjene :exploding_head: